src/app/public/modules/sort/sort.component.ts

```diff
@@ -39,6 +45,18 @@ export class SkySortComponent {
 
   public sortByHeadingId: string = `sky-sort-heading-${++nextId}`;
 
+  constructor(
+    public themeSvc: SkyThemeService,
+    private changeDetector: ChangeDetectorRef
+  ) {
+  }
+
+  public ngOnInit(): void {
+    this.themeSvc.settingsChange.subscribe(() => {
+      this.changeDetector.markForCheck();
```

Comment:
This addresses child components that use `ChangeDetectionStrategy.OnPush` so that they pick up theme changes. Without it there is a delay in e.g. icon changes until there is another change pushed.